rubrics/impl/src/main/java/org/sakaiproject/rubrics/logic/RubricsServiceImpl.java

```diff
@@ -312,14 +312,18 @@ public void saveRubricAssociation(String tool, String id, Map<String,String> par
                     String input = "{\"toolId\" : \""+tool+"\",\"itemId\" : \"" + id + "\",\"rubricId\" : " + params.get(RubricsConstants.RBCS_LIST) + ",\"metadata\" : {\"created\" : \"" + created + /*"\",\"modified\" : \"" + nowTime +*/ "\",\"ownerId\" : \"" + owner +
 					"\",\"ownerType\" : \"" + ownerType + "\",\"creatorId\" : \"" + creatorId + "\"},\"parameters\" : {" + setConfigurationParameters(params,oldParams) + "}}";
                     log.debug("Existing association update" + input);
+                    String [] assocSplitted = associationHref.split("/");
+                    deleteRubricEvaluationsForAssociation(assocSplitted[assocSplitted.length-1], tool);
```

Comment:
would it make sense to check the array length here, assocSplitted.length >= 1, or are you sure it'll be okay?

Reply:
I'm pretty sure it will be ok, there are similar split process in the service, plus I checked there's a valid associationHref object everywhere I add it. Anyway, as I'm going to put it inside the method I can add that verification.